src/options.h

```diff
@@ -843,15 +843,15 @@ struct option_map_value
    argtype_e          type;
    int                min_val;
    int                max_val;
-   const char         *name;
-   const char         *short_desc;
-   const char         *long_desc;
+   string             name;
```

Comment:
It is not really needed to work with strings here but it makes everything so much easier when one can avoiding pointers in emscripten.